src/Domain/GameManager.cs

```diff
@@ -21,12 +21,7 @@ public GameManager(IGameRepository gameRepository, IMoveRepository moveRepositor
 
         public async Task<Game> CreateGame(CancellationToken cancellationToken)
         {
-            Game g = new Game
-            {
-                Id = Guid.NewGuid(),
-                Winner = null,
-                FinishedAt = null,
-            };
+            Game g = new Game {Id = Guid.NewGuid(), Winner = null, FinishedAt = null,};
```

Comment:
Should we use something that is shorter than Guid that a human a can type in a reasonable time?
Something like: PPBqWA9 (https://github.com/dylang/shortid, this is not in C# though)

Reply:
Yes later. Created Issue #7 for that